software/htmly.yml

```diff
@@ -0,0 +1,14 @@
+name: "HTMLy"
+website_url: "https://www.htmly.com/"
+source_code_url: "https://github.com/danpros/htmly/"
+description: "HTMLy is an open source databaseless PHP blogging platform. A flat-file CMS that allows you to create a fast, secure, and powerful website or blog in seconds."
+licenses:
+  - GPL-2.0
+platforms:
+  - PHP
+tags:
+  - Blogging Platforms
+  - Content Management Systems (CMS)
+depends_3rdparty: false
```

Comment:
```suggestion
```
This is a redundant line, its already set to false 👍🏻

Reply:
You are right. I didn't understand the explanation in the template.